open-api/src/testFixtures/java/org/apache/iceberg/rest/RESTCatalogServer.java

```diff
@@ -64,7 +65,9 @@ public Map<String, String> configuration() {
 
   private CatalogContext initializeBackendCatalog() throws IOException {
     // Translate environment variables to catalog properties
-    Map<String, String> catalogProperties = RCKUtils.environmentCatalogConfig();
+    Map<String, String> catalogProperties = Maps.newHashMap();
+    catalogProperties.putAll(RCKUtils.environmentCatalogConfig());
+    catalogProperties.putAll(Maps.fromProperties(System.getProperties()));
```

Comment:
as mentioned in a comment further above, maybe we should consider passing a Map<String, String> to `RESTCatalogServer` rather than relying on system properties (which have to be cleared after they were configured)

Reply:
Done.

Reply:
that means we can revert this line here: `catalogProperties.putAll(Maps.fromProperties(System.getProperties()));`

Reply:
Reverted.